internal/kafka/command_cluster_create.go

```diff
@@ -171,6 +172,17 @@ func (c *clusterCommand) create(cmd *cobra.Command, args []string) error {
 		setClusterConfigCku(&createCluster, int32(cku))
 	}
 
+	if cmd.Flags().Changed("network") {
+		networkId, err := cmd.Flags().GetString("network")
```

Comment:
We've found that it's a bit easier to debug when you give variables the same name as flags
```suggestion
		network, err := cmd.Flags().GetString("network")
```